plugins/modules/pnp_intent.py

```diff
@@ -597,12 +614,13 @@ def get_have(self):
 
                 if site_exists:
                     have["site_id"] = site_id
-                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id))
-                    self.log("Site Name:" + str(site_name))
+                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id), "INFO")
+                    self.log("Site Name:" + str(site_name), "INFO")
```

Comment:
Not required, we can club it in the before log message.

Reply:
Done